docs/install_guides/nci_gadi/install_gadi.sh

```diff
@@ -1,5 +1,36 @@
 #!/bin/bash
 
+usage="
+Usage:
+  A script to install and run an Underworld software stack.
+
+** To install **
+Review script details: modules, paths, repository urls / branches etc.
+ $ source <this_script_name>
+ $ install_full_stack
+see script for details, running $ install_petsc maybe required.
+
+** To run **
+Add the line
+ source <this_script_name>
```

Comment:
What's the story here @julesghub?  

So they have to run `source install_gadi.sh` to run UW?

Reply:
yep - that's the idea